packages/ra-ui-materialui/src/layout/TitlePortal.stories.tsx

```diff
@@ -28,3 +28,10 @@ export const Sx = () => (
         <Title title="Hello, world" />
     </TestMemoryRouter>
 );
+
+export const NonConfigurable = () => (
```

Comment:
**suggestion:** This behavior should be tested in the context a `<DatagridConfigurable>` component since that's the goal of this PR.

Reply:
Sorry, it was not clear. The goal is to test that the title is no longer configurable in "configurable mode" (not necessarily in a `<DatagridConfigurable />` context) with the prop you added. 
To achieve this, you can take inspiration from `react-admin/packages/ra-ui-materialui/src/preferences/Configurable.stories.tsx`. You can create a new story in this file and run storybook with `make storybook` to see the result 🙂

Reply:
Agreed, the story must be in a Configurable context, and show that when switching to configurable mode, the title isn't editable.

Reply:
Just pushed this change. Lmk if you see any issues :)